### DISCUSSION TITLE
用pyarmor和nuitka兼容吗?

### DISCUSSION THREAD

**Discussion by fxy055** [dashingsoft/pyarmor#1945] [Q&A]
我用pyarmor试用版加密后再用nuitka打包后程序不可执行，少了多文件。

**Comment by fxy055** [comment]
![Snipaste_2024-10-07_23-11-46](https://github.com/user-attachments/assets/0e06f10d-31ea-4a82-8e39-a550e0873e41)

![Snipaste_2024-10-07_23-11-50](https://github.com/user-attachments/assets/c710b43a-86de-4152-a3a8-8dfd0b119be9)

**Comment by jondy** [comment]
对使用 nuitka 打包不太清楚，但是如果 nuitka 需要分析脚本去发现依赖模块的话，那应该是因为加密之后无法找到依赖模块了
